app/serializers/folder_minimal_serializer.rb

```diff
@@ -4,4 +4,10 @@ class FolderMinimalSerializer < ApplicationSerializer
   attributes :id, :name, :description
 
   has_many :encryptables, serializer: EncryptableMinimalSerializer
+
+  def encryptables
+    object.encryptables.reject do |encryptable|
```

Comment:
now I know what you meant ... looks like a ok solution.
problem is that all encryptable files are fetched from db this way.
but the idea is to move files anyway to same layer as credentials, ose secrets are.
so we leave it that way for now.